src/actions/sources/loadSourceText.js

```diff
@@ -9,16 +9,17 @@ import { PROMISE } from "../utils/middleware/promise";
 import { getSource, getGeneratedSource } from "../../selectors";
 import * as parser from "../../workers/parser";
 import { isLoaded } from "../../utils/source";
+import { Telemetry } from "devtools-modules";
 
 import defer from "../../utils/defer";
 import type { Action, ThunkArgs } from "../types";
 import type { SourceRecord } from "../../types";
 
 const requests = new Map();
-import { Services } from "devtools-modules";
-const loadSourceHistogram = Services.telemetry.getHistogramById(
-  "DEVTOOLS_DEBUGGER_LOAD_SOURCE_MS"
-);
+
+const loadSourceHistogram = "DEVTOOLS_DEBUGGER_LOAD_SOURCE_MS";
+const telemetry = new Telemetry();
+telemetry.start(loadSourceHistogram);
```

Comment:
hmm, what is this trying to measure?

it might be capturing the time from module load to the completed load source...

also, it'd be good to be careful that you don't deal w/ race conditions...

like three sources loading and the 2nd finishing first

Reply:
The probes are in the same place as they previously were but starting on module load seems wrong to me as well... I will try to find a better place.